src/test/java/ch/ivyteam/ivy/maven/test/TestSetupIvyTestPropertiesMojo.java

```diff
@@ -66,7 +66,7 @@ public void engineModuleHintsSharedAsProperty() throws Exception {
 
     mojo.execute();
 
-    assertThat(getProperty(Property.MAVEN_TEST_ARGLINE)).contains(" --add-opens ");
+    assertThat(getProperty(Property.MAVEN_TEST_ARGLINE)).contains(" --add-opens=");
```

Comment:
before the --add-opens / --add-export properties where added as `--add-opens blablabla` and now `--add-opens=blablabla` is there an difference?

Reply:
I think the JVM args for surefire work exactly like any other argument for a normal 'java' binary. So if `--add-opens xyz` and `--add-opns=XYZ` are both interpreted by the JVM I'm fine with the change